src_c/image.c

```diff
@@ -64,6 +53,15 @@ opengltosdl(void);
     ((flipped) ? (((char *)data) + (height - row - 1) * width) \
                : (((char *)data) + row * width))
 
+static PyObject *extloadobj = NULL;
+static PyObject *extsaveobj = NULL;
+static PyObject *extverobj = NULL;
+
+/* define docs for undocumented (kinda deprecated) functions */
+#define DOC_PYGAMEIMAGELOADBASIC "internal function to load images, use pygame.image.load() instead of this"
```

Comment:
This docstrings should be in this file: https://github.com/pygame/pygame/blob/master/src_c/doc/image_doc.h

Reply:
Yeah, but these are temporary-undocumented functions, they cannot go in that file right.

Reply:
Plus those files are auto generated from the rest files, I guess they will dissapear the next time, those files get generated.